### DISCUSSION TITLE
Using record id for storage path in insert form

### DISCUSSION THREAD

**Discussion by rezakho** [filamentphp/filament#11315] [Help]
### Package

Form builder

### Package Version

v3

### How can we help you?

How to use record id for storage path in insert form

**Comment by leandrocfe** [comment]
https://filamentphp.com/plugins/filament-spatie-media-library
?

**Comment by rezakho** [comment]
Sorry! I don't find the solution!
There is no id before model saving, so i think a method may be exists that moving file after saving or what?!
